sql_generators/mobile_kpi_support_metrics/templates/new_profile_clients.view.sql

```diff
@@ -0,0 +1,34 @@
+{{ header }}
+CREATE OR REPLACE VIEW
+  `{{ project_id }}.{{ dataset }}.{{ name }}`
+AS
+SELECT
+  client_id,
+  first_seen_date,
+  normalized_channel,
+  app_name,
+  app_display_version AS app_version,
+  country,
+  locale,
+  isp,
+  normalized_os AS os,
+  normalized_os_version AS os_version,
+  device_model,
+  device_manufacturer,
+  is_mobile,
+  {% for attribution_field in product_attribution_fields.values() if not attribution_field.name.endswith("_timestamp") %}
+  attribution.{{ attribution_field.name }},
+  {% endfor %}
+  {% if 'adjust_network' in product_attribution_fields %}
+    `moz-fx-data-shared-prod.udf.organic_vs_paid_mobile`(adjust_network) AS paid_vs_organic,
+  {% else %}
+    "Organic" AS paid_vs_organic,
+  {% endif %}
```

Comment:
[in the spec ](https://docs.google.com/document/d/13aSkbugFFKH0e68uPmNzawtnvG5ShtRjCXoDhz2doH4/edit) we also had `install_source`. its historical source is the metrics ping, but apparently it was recently added to the baseline ping and is riding the trains. I don't know if it is present in attribution_clients - if it isn't maybe we should discuss how important it is. I do think that @soGaussian does use it commonly.

Reply:
I do not use the `install_source` field regularly, I think it being on the clients table is good enough for the rare use cases.

Reply:
This should be part of the attribution field loop and should be included. I just double checked as far as I can see it is showing up for Fenix:
<img width="503" alt="image" src="https://github.com/user-attachments/assets/e4399f3e-bad9-41fd-9ee9-f568c437196e">

Reply:
Ah, got it. thanks for confirming!